reference-adapter/pspcl-ifix-adapter/src/main/java/org/egov/util/RequestHeaderUtil.java

```diff
@@ -0,0 +1,35 @@
+package org.egov.util;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+import org.egov.common.contract.request.RequestHeader;
+import org.egov.common.contract.request.UserInfo;
+import org.egov.config.PspclIfixAdapterConfiguration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+import java.util.UUID;
+
+
+@Component
+@NoArgsConstructor
+@AllArgsConstructor
+public class RequestHeaderUtil {
+
+    @Autowired
+    private PspclIfixAdapterConfiguration adapterConfiguration;
+
+    public RequestHeader getRequestHeader() {
+        RequestHeader requestHeader = new RequestHeader();
+        UserInfo userInfo = new UserInfo();
+        userInfo.setUuid(UUID.randomUUID().toString());
```

Comment:
Why are you trying to set these? This will get set at the API gateway when the fiscal event request contains a valid bearer token.

Reply:
i'll remove.